doc/nrf/protocols/thread/certification.rst

```diff
@@ -177,3 +180,48 @@ This is valid only for Nordic Semiconductor development kits with a J-Link virtu
 
 To add an nRF52840 DK, drag the nRF52840 DK and drop it on the test bed configuration page.
 After that, the device is configured and the :ref:`proper baud rate (115200) <test_and_optimize>` and COM port are set.
+
+.. _ug_thread_build_report:
+
+OpenThread build report
+***********************
+
+The OpenThread build report contains information about the current |NCS| and OpenThread revisions, Thread feature set, Thread library path and changes in the :ref:`nrfxlib:nrfxlib` repository in comparison with the latest |NCS| release.
+The report is generated to the output console log, and stored as an additional build artefact in the application build directory.
+
+To enable building the Thread report set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT` kconfig option to ``y``.
+To disable generating the build artefact set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT` kconfig option to ``n``.
+
+By default the report artefact name is set to the :file:`ot_report.txt`, but you can specify your own name by setting the :kconfig:option:`CONFIG_OPENTHREAD_REPORT_BUILD_ARTEFACT_NAME` kconfig value to the new one.
+
+While building the application using the :ref:`nrfxlib:ot_libs` you will see the following logs in your build console:
+
+.. code-block::
+
+   ################### OPENTHREAD REPORT ###################
+   + Target device: nrf52840
+   + Thread version: v1.3
+   + Feature set: master
+   + OpenThread Library from nrfxlib: openthread/lib/cortex-m4/soft-float/v1.3/master/
+   + OpenThread NCS revision: thread-reference-20230706-755-ga759a4a09
+   + NCS revision: v2.7.0-882-g3097ca5d36
+   ###################        END        ###################
+
+The generated build artefact will also include the list of changes in the nrfxlib directory.
+If there are no changes related to the used Thread library it means that you can use :ref:`ug_thread_cert_inheritance_without_modifications`.
+Any changes in the used Thread library that were detected and shown in the report you will need to contact Thread Group to check if certification by inheritance is still possible.
+
+While building the application and Thread stack from its source files the OpenThread report will be different and you will see the following logs in your build console:
```

Comment:
```suggestion
When building the application and Thread stack from its source files, the OpenThread build report will be different and you will see the following logs in your build console:
```
Another option here is to have the two different ways of building as tabs, to show more clearly that they're options.

Reply:
Yeah, I thought about it, I will add it here because indeed it will be better.

Reply:
I've added it, let's see what it will look like after the doc build.